pom.xml

```diff
@@ -145,5 +145,4 @@
         </repository>
 
     </repositories>
-
```

Comment:
At first I tried it in the pom.xml which is in the folder of the binding. The second try was with the pom.xml in the root-folder of the openhab2-addons. Still nojoy??